src/app/components/accountRow/index.tsx

```diff
@@ -162,13 +162,19 @@ function AccountRow({
     navigator.clipboard.writeText(account?.btcAddress!);
     setOnBtcCopied(true);
     setOnStxCopied(false);
-    if (showBtcReceiveAlert !== null) { dispatch(ChangeShowBtcReceiveAlertAction(true)); }
+    // set 'Copied' text back to 'Bitcoin address' after 3 seconds
+    setTimeout(() => setOnBtcCopied(false), 3000);
```

Comment:
We should also clear the timeout when the component unmounts.

Reply:
Added that in this commit https://github.com/secretkeylabs/xverse-web-extension/pull/424/commits/14ae91a94adebf2f61820dd7d5687f860732bc5d